eng/common/pipelines/templates/steps/credscan.yml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  SuppressionFilePath: 'eng\CredScanSuppression.json'
+  BaselineFilePath: ''
+steps:
+- pwsh: |
+    New-Item -Path "$(Build.sourcesdirectory)" -Name "credscan.tsv"
+    if ("$(Build.Reason)" -eq 'PullRequest') {
+      New-Item -Path "$(Build.sourcesdirectory)" -Name "changedFiles" -ItemType "directory"
+      (git diff "origin/$(System.PullRequest.TargetBranch)" HEAD --name-only)
+      | ForEach-Object { Add-Content -Path "$(Build.SourcesDirectory)\credscan.tsv" -Value "$(Build.SourcesDirectory)\$_"}
+    }
+    else {
+      Set-Content "credscan.tsv" -Value "$(Build.SourcesDirectory)\sdk\${{ parameters.ServiceDirectory }}"
```

Comment:
```suggestion
      Set-Content "credscan.tsv" -Value "$(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}"
```

Reply:
It looks like you also need to declare the ServiceDirectory parameter.